config/routes.rb

```diff
@@ -12,4 +12,5 @@
   resources :posts, only: [:show, :new, :create]
 
   get 'users/:nickname', to: 'users#show', as: 'show_user', nickname: /[^\/]+/
+  get 'users', to: 'users#index', as: 'users'
```

Comment:
You can use the method `resources`, just like you did with posts. You'd only be interested in the index endpoint.